Shared/Samples/Display clusters/DisplayClustersView.swift

```diff
@@ -71,7 +86,26 @@ struct DisplayClustersView: View {
                 ) { [popup] in
                     PopupView(popup: popup!, isPresented: $showsPopup)
                         .showCloseButton(true)
-                        .padding()
+                        .padding([.top, .horizontal])
+                    
+                    if !geoElements.isEmpty {
+                        List {
+                            Section {
+                                ForEach(Array(geoElements.enumerated()),
+                                        id: \.offset
```

Comment:
When I right click "offset" to show quick help, it links to an irrelevant doc. Maybe you can make a Apple Developer forum post.